### DISCUSSION TITLE
Install spezific Kubernetes Version

### DISCUSSION THREAD

**Discussion by Delta1977** [siderolabs/talos#7867] [Q&A] [answered]
I search in the doku but found no hint:
In Talos 1.5 Kubernetes versions1.28, 1.27, 1.26 are support.

How do i install Talos with Kubernetes 1.26 ?

**Comment by smira** [comment, marked as answer]
e.g. `talosctl gen config --kubernetes-version=1.26.x`

Kubernetes component version is part of Talos machine configuraiton.

**Comment by Delta1977** [comment]
Thank you for the quick answer
